app/models/miq_event.rb

```diff
@@ -24,7 +24,7 @@ def self.raise_evm_event(target, raw_event, inputs = {}, options = {})
     # Policy, automate, and alerting could then consume this type field along with the details
     if target.kind_of?(Array)
       klass, id = target
-      klass = Object.const_get(klass)
+      klass = klass.constantize
       target = klass.find_by_id(id)
```

Comment:
You can reduce this to: `target = klass.constantize.find_by(:id => id)` and get rid of the `klass = ` line.